osbs/build/config_map_response.py

```diff
@@ -34,10 +35,23 @@ def get_data(self):
         """
         Find the data stored in the config_map
 
-        :return: dict, the json data that was passed into the ConfigMap on creation
+        :return: dict, the data that was passed into the ConfigMap on creation
         """
         data = graceful_chain_get(self.json, "data")
         if data is None:
             return {}
 
         return data
+
+    def get_json(self):
+        """
+        Find the data stored in the config_map
+
+        :return: json, the json version of the json data that was passed into the ConfigMap on
+                 creation
+        """
+        data = graceful_chain_get(self.json, "data")
+        if data is None:
+            return {}
+
+        return json.dumps(data)
```

Comment:
I think we want `loads` here, i.e.:
"""
Return the object stored in the config map

:param name: str, key name
:return: object, the object described by a JSON string at key 'name'
""""

data = ...

return json.loads(data[name])